### PR TITLE
Shotgun fun

### PR DESCRIPTION
**Pull request by Atskadan**
Adds 5 new styles to the barman's shotguns, and allows the bartender to rename their gun. 
ss 2014-03-12 at 10 47 01
ss 2014-03-12 at 10 47 13
From left to right:
Standard Walnut (original), Dark Red Finish, Ash, Faded Grey, Maple, and Rosewood.

Sprite credit to Stephie! Thank you.

### CODE REVIEW

**Comment by MrSnapwalk** [comment]
The bartender should be able to rename his gun as well. Otherwise, two thumbs up!
EDIT: I am a derp, and I did not read the entire description. Carry on!

File: code/modules/projectiles/guns/projectile/shotgun.dm [hunk 1]
```diff
@@ -69,28 +68,60 @@
...
+	var/mob/M = usr
+	var/input = stripped_input(M,"What do you want to name the gun?", ,"", MAX_NAME_LEN)
+
+	if(src && input && !M.stat && in_range(M,src))
+		name = input
+		M << "You name the gun [input]. Say hello to your new friend."
+		return 1
+
+/obj/item/weapon/gun/projectile/revolver/doublebarrel/verb/reskin_gun()
+	set name = "Change gun furniture"
```
Comment (RemieRichards) [inline code/modules/projectiles/guns/projectile/shotgun.dm:87]: Furniture? don't you meant furnishings? Apparently they can mean the same thing but I feel furnishing makes a lot more sense.

**Comment by StarToad** [comment]
Eh, it'd be easy to fix within github's code editor, but that's up to Miggles. If he wants it to stay that way so be it. I'll merge it after he says his opinion on the matter of spelling.

File: code/modules/projectiles/guns/projectile/shotgun.dm (hunk 1 above)
Comment (Atskadan) [inline code/modules/projectiles/guns/projectile/shotgun.dm:87]: It's called furniture.

**Comment by Atskadan** [comment]
It's furniture. If a gun has wooden furniture, it means the stock and grips are made of wood.

**Comment by StarToad** [comment]
Alright, it's going in. Pretty neat spritework, too.

**Pull request merged**